extensions/wikia/AdEngine/js/AdLogicPageParams.js

```diff
@@ -1,21 +1,20 @@
 /*jshint camelcase:false*/
 /*global define, require*/
 define('ext.wikia.adEngine.adLogicPageParams', [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.adEngine.adLogicPageViewCounter',
```

Comment:
Thanks for alphabetical ordering.

I think I used to put wikia.* modules before ext.wikia.adEngine.* though, but let's keep it like this.